modules/mesh-task/main.tf

```diff
@@ -115,11 +237,6 @@ resource "aws_ecs_task_definition" "this" {
                 hostPort      = 8300
                 protocol      = "udp"
               },
-              {
-                containerPort = 8500
-                hostPort      = 8500
-                protocol      = "tcp"
-              },
```

Comment:
Removing since nobody should need to talk to the consul client outside the mesh task and so it should expose its http port on the host.